internal/apischema/openai/openai.go

```diff
@@ -184,6 +184,15 @@ func (s *StringOrArray) UnmarshalJSON(data []byte) error {
 		return nil
 	}
 
+	// Try to unmarshal as array of strings (for embeddings)
```

Comment:
super nit, but could you add period for all code comments not only here?

```suggestion
	// Try to unmarshal as array of strings (for embeddings).
```